portal-web/test/functional/com/liferay/portalweb/macros/LexiconEntry.macro

```diff
@@ -1,5 +1,22 @@
 definition {
 
+	macro changeCurrentDisplayStyle {
+		WaitForPageLoad();
+
+		if (IsElementNotPresent(locator1 = "MenuItem#ANY_MENU_ITEM")) {
+			Refresh();
+
+			Click.mouseOverClick(locator1 = "Icon#CURRENT_DISPLAY_STYLE");
+		}
+		else {
```

Comment:
@koor4356 What is the point of this fallback? It just does a different click on the same locator